wazimap_np/templates/profile/profile_detail.html

```diff
@@ -7,14 +7,5 @@
 {% endblock %}
 
 {% block profile_detail %}
-    {% include 'profile/sections/demographics.html' %}
-    {% include 'profile/sections/elections.html' %}
-    {% include 'profile/sections/education.html' %}
-    {% include 'profile/sections/health.html' %}
-    {% include 'profile/sections/households.html' %}
-    {% include 'profile/sections/business.html' %}
-    {% include 'profile/sections/agriculture.html' %}
-    {% include 'profile/sections/forests.html' %}
-    {% include 'profile/sections/disasters.html' %}
-    {% include 'profile/sections/development.html' %}
+    <!-- TODO: do we keep the block? needs something here? -->
```

Comment:
The federal instance will almost certainly have its HTML broken into profile sections.